DLCS.Repository/Assets/AssetRepository.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Dapper;
+using DLCS.Model.Assets;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Npgsql;
+
+namespace DLCS.Repository.Assets
+{
+
+    public class AssetRepository : IAssetRepository
+    {
+        private readonly IConfiguration configuration;
+        private readonly ILogger<AssetRepository> logger;
+
+        public AssetRepository(IConfiguration configuration, ILogger<AssetRepository> logger)
+        {
+            this.configuration = configuration;
```

Comment:
Use [options pattern](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-3.1) rather than pass around `IConfiguration`?

Reply:
Again - this slice through a potential arrangement of the stack using a very simple service allows us to decide our patterns and practices on something manageable.